examples/step-29/doc/results.dox

```diff
@@ -43,33 +43,39 @@ parameters here and therefore stick with their default values.
 Here's the console output of the program in debug mode:
 
 @code
-examples/\step-29> make run
-============================ Running \step-29
-DEAL::Generating grid... done (1.11607s)
+$ make run
+[ 66%] Built target step-29
+[100%] Run step-29 with Debug configuration
+DEAL::Generating grid... done (0.832682s)
 DEAL::  Number of active cells:  25600
-DEAL::Setting up system... done (1.10807s)
+DEAL::Setting up system... done (0.614761s)
 DEAL::  Number of degrees of freedom: 51842
-DEAL::Assembling system matrix... done (6.50841s)
-DEAL::Solving linear system... done (2.89218s)
-DEAL::Generating output... done (4.52428s)
+DEAL::Assembling system matrix... done (2.82536s)
+DEAL::Solving linear system... done (2.27971s)
+DEAL::Generating output... done (1.84418s)
```

Comment:
It's pretty interesting to see how different components (setup, assembly, solver) have evolved differently in terms of their time (and your system versus the one originally used).